README.md

```diff
@@ -39,28 +39,87 @@ Same as the original conductor, the primary goals of this project are to...
 - Provide a free to use framework for any starting enterprise, or individual programmer.
 - Automatic detection of connected iOS devices
 
-# Default Properties
+# Configuration
+Conductor can be configured using a [yaml](https://en.wikipedia.org/wiki/YAML) file. By default, Conductor looks for a "config.yaml" at the root of embedded resources.
+
+The file has 3 sections: `current configuration`, `defaults`, and `schemes`.
+
+## General Configuration
+The general configuration only has two properties: `platformName` (which must be either IOS or ANDROID) and optionally `currentSchemes`, which will be discussed later
```

Comment:
Lowercase these to `ios` and `android`

Reply:
No because of the way we're parsing the enum, they have to be all uppercase :(